contrib/hello-world-react/src/server.js

```diff
@@ -0,0 +1,43 @@
+import path from 'path';
+import http from 'http';
+import express from 'express';
+import bodyParser from 'body-parser';
+
+export default function(config) {
+
+	const app = express();
+	const server = http.createServer(app);
+
+    let products = [
+        {id: '1', name: 'iphone 6', cost: 300, quantity: 0},
```

Comment:
The quantities here should be obsolete by the changes we discussed earlier, right? You should never be accessing/changing them anymore.